enterprise/glassfish.common/nbproject/org-netbeans-modules-glassfish-common.sig

```diff
@@ -620,6 +635,8 @@ CLSS public abstract interface java.util.EventListener
 
 CLSS public abstract interface java.util.Map<%0 extends java.lang.Object, %1 extends java.lang.Object>
 innr public abstract interface static Entry
+meth public !varargs static <%0 extends java.lang.Object, %1 extends java.lang.Object> java.util.Map<{%%0},{%%1}> ofEntries(java.util.Map$Entry<? extends {%%0},? extends {%%1}>[])
+ anno 0 java.lang.SafeVarargs()
 meth public abstract boolean containsKey(java.lang.Object)
```

Comment:
@ebarboni @matthiasblaesing why are there JDK classes in the sig file of this module?

Reply:
the sig generator is probably not filtering out JDK classes, which means if the public API uses JDK classes in declarations, the sig changes when JDK level changes.